pkg/security/ldapclient/client.go

```diff
@@ -57,6 +59,43 @@ type ldapClientConfig struct {
 // ldapClientConfig is an Config
 var _ Config = &ldapClientConfig{}
 
+// ldapDial is the same as ldap.Dial except that it uses a proxyDialer that respects
```

Comment:
Could we have also point out in the docs of both these functions that while the upstream allows supplying a custom dialer by `ldap.DialWithDialer`, this cannot currently be used because it specifically requires the `*net.Dialer` struct?